src/main/java/org/casbin/jcasbin/util/StringPool.java

```diff
@@ -0,0 +1,86 @@
+/*
+ *      Copyright (c) 2018-2028, DreamLu All rights reserved.
+ *
+ *  Redistribution and use in source and binary forms, with or without
+ *  modification, are permitted provided that the following conditions are met:
+ *
+ *  Redistributions of source code must retain the above copyright notice,
+ *  this list of conditions and the following disclaimer.
+ *  Redistributions in binary form must reproduce the above copyright
+ *  notice, this list of conditions and the following disclaimer in the
+ *  documentation and/or other materials provided with the distribution.
+ *  Neither the name of the dreamlu.net developer nor the names of its
+ *  contributors may be used to endorse or promote products derived from
+ *  this software without specific prior written permission.
+ *  Author: DreamLu 卢春梦 (596392912@qq.com)
+ */
+package org.casbin.jcasbin.util;
+
+/**
+ * 静态 String 池
+ *
+ * @author L.cm
+ */
+public interface StringPool {
```

Comment:
I think we don't need this file, just using "," as comma would be OK.

Reply:
Thanks for your advice, I will deal with it.